.github/workflows/publish-artifacts.yml

```diff
@@ -176,8 +178,10 @@ jobs:
 
     - name: Publish CDH + ASR with ORAS
       id: publish
+      env:
+       OCI_ARCH: ${{ matrix.arch == 'powerpc64le' && 'ppc64le' || matrix.arch }}
```

Comment:
The logic here is sound. Is it possible to express it in the same way as AA? s.t.
no `env`, but set `arch_tag`, e.g.
```
arch="${{ matrix.arch }}"
[ "$arch" = "powerpc64le" ] && arch="ppc64le"
tag="${{ github.sha }}-${arch}"
```

Reply:
It makes sense, modified it, Thanks @Xynnn007! PTAL